nodle-testnet.yaml

```diff
@@ -0,0 +1,43 @@
+specVersion: 1.0.0
+name: subql-nodle
+version: 1.0.0
+runner:
+  node:
+    name: '@subql/node'
+    version: '*'
+  query:
+    name: '@subql/query'
+    version: '*'
+description: 'Subquery indexer for Nodle network (testnet).'
+repository: https://github.com/NodleCode/subquery.git
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '0x0fd167da222f10dae826a0433fe4daadcf4a36c2a6bd5c1b085857d5b79030af'
+  endpoint: wss://nodle-paradis.api.onfinality.io/public-ws
+  chaintypes:
+    file: ./dist/chaintypes.js
+dataSources:
+  - kind: substrate/Runtime
+    startBlock: 1
+    mapping:
+      file: ./dist/index.js
+      handlers:
+        - handler: handleBlock
+          kind: substrate/BlockHandler
+        - handler: systemTokenTransferEvent
+          kind: substrate/EventHandler
+          filter:
+            module: balances
+            method: Transfer
+        - handler: handleVestingScheduleAdded
+          kind: substrate/EventHandler
+          filter:
+            module: vesting
+            method: VestingScheduleAdded
+        - handler: handleAllocation
+          kind: substrate/EventHandler
+          filter:
+            module: allocations
+            method: newallocation
```

Comment:
Are we going to change this?

Reply:
It's updated now. Tested the current deployed versions  and all is fine